CMakeLists.txt

```diff
@@ -11,7 +18,7 @@ endif ()
 # for Windows only
 if (WIN32)
   set(CMAKE_CXX_FLAGS_RELEASE "-DNOMINMAX ${CMAKE_CXX_FLAGS_RELEASE} /MT")
-  set(CMAKE_CXX_FLAGS_DEBUG "-DNOMINMAX ${CMAKE_CXX_FLAGS_DEBUG} /MTd")
+  set(CMAKE_CXX_FLAGS_DEBUG "-DNOMINMAX ${CMAKE_CXX_FLAGS_DEBUG} /MDd")
```

Comment:
Add some documentation as to why the switch needed to be changed.

Reply:
Yeah, this was added long time ago for Gtest experiments works, and they might changed / fixed the machanism. So you can try to remove those 2 above lines to see if it work or not, if it work then we can simply just comment and document it.